allwinner-hal/src/smhc/structure.rs

```diff
@@ -8,6 +8,164 @@ use crate::ccu::{self, Clocks, SmhcClockSource};
 use core::arch::asm;
 use embedded_sdmmc::{Block, BlockDevice, BlockIdx};
 
+#[repr(C)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Default)]
+struct IDMACDescriptor {
+    pub des0: IDMACDescriptor0,
+    pub des1: IDMACDescriptor1,
```

Comment:
```suggestion
struct IdmacDescriptor {
    pub des0: IdmacDescriptor0,
    pub des1: IdmacDescriptor1,
```

Is this capitalization more Rusty?

Reply:
Reasonable, thanks and updated it.